### PR TITLE
patch Hyperopt to make it work with networkx 2.0

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

**Comment by smoors** [comment]
@wpoely86 review?

File: easybuild/easyconfigs/h/Hyperopt/Hyperopt-0.1-intel-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -12,7 +12,12 @@ toolchain = {'name': 'intel', 'version': '2017b'}
 
 source_urls = [PYPI_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['4f6e903f7640165ea3e4c622050b41ffab0bee7811ede23c7825a5884976d72f']
+patches = ['Hyperopt-0.1_networkx-2.0.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/h/Hyperopt/Hyperopt-0.1-intel-2017b-Python-2.7.14.eb]: @smoors Please avoid hardcoding the software version here if it matches: `Hyperopt-%(version)s_networkx-2.0.patch` (and drop the empty line below while you're at it)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d86cb72cc9f3c048fba15af86b8e62e6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/de8de4aa64f28c6db417e6fb6adce366 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**